File: Functions/prime.cpp
#include<iostream>
using namespace std;

int main(){
    int n;
    cout<<"Enter a number: ";
    cin>>n;
    bool isPrime = true;

    if(n<=1){
        isPrime = false;
    }
    else{
        for(int i=2;i<=n/2;i++){
            if(n%i==0){
                isPrime = false;
                break;
            }
        }
    }
    if(isPrime){
        cout<<"The number is prime."<<endl;
    }
    else{
        cout<<"The number is not prime."<<endl;
    }
}

File: java/prime.java

class prime {
    public static void main(String[] args) {
        int num = 5;
    
        for(int i=2;i<num;i++){
            if(num % i == 0){
                System.err.println("The number is not prime");
                break;
            }

           
        }

         System.err.println("The number is prime");
    }
}
